info-orbs.code-workspace

```diff
@@ -14,7 +14,7 @@
     "[jsonc]": {
       "editor.defaultFormatter": "esbenp.prettier-vscode"
     },
-    "editor.formatOnSave": true
```

Comment:
I'm going to reapply this setting when we merge the formatter PR https://github.com/brettdottech/info-orbs/pull/131